### PR TITLE
Generate OLM manifests

### PR DESCRIPTION
**Pull request by razvan**
Fixes https://github.com/stackabletech/stackable-utils/issues/52

### CODE REVIEW

**Review by adwk67 — CHANGES_REQUESTED** [review]

File: olm/build-manifests.sh [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+#
+#   ./olm/build-manifests.sh -r <release as x.y.z> -p <product> -c <location of the RH cert operators repo> -o <location of the oeprator repo>
+#
+# Example:
+#
+#   ./olm/build-manifests.sh -r 23.11.0 -c $HOME/repo/stackable/openshift-certified-operators -o $HOME/repo/stackable/zookeeper-operator
+#
+# The generated manifests need to be updated manually with the following steps:
+# * Copy a clientserviceversion file from previous version and update the cluster role and the deployment
+# * Insert the contents of the deployment, cluster role and service account that are
```
[the remark this answers is not in the document]
Comment (adwk67) [reply olm/build-manifests.sh:15]: Ah, I've had another look and seen that you've replaced the in-line cluster permissions with ones covered by the cluster role.

[the remark this answers is not in the document]
Comment (adwk67) [reply olm/build-manifests.sh:15]: Maybe we could document this schematically with placeholders: this-bit-goes-here etc.

File: olm/build-manifests.sh [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+#   generated by Helm.
+# * Remove unused files that are included in the CSV: service account, cluster role, deployment.
+# * Remove all Helm labels in all remaining files.
+# * Check or update the metadata/dependecies.yaml
```
Comment (adwk67) [inline olm/build-manifests.sh:19]:
```suggestion
# * Check or update the metadata/dependencies.yaml
```

File: olm/build-manifests.sh [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+#
+# Usage:
+#
+#   ./olm/build-manifests.sh -r <release as x.y.z> -p <product> -c <location of the RH cert operators repo> -o <location of the oeprator repo>
+#
+# Example:
+#
+#   ./olm/build-manifests.sh -r 23.11.0 -c $HOME/repo/stackable/openshift-certified-operators -o $HOME/repo/stackable/zookeeper-operator
+#
+# The generated manifests need to be updated manually with the following steps:
```
[the remark this answers is not in the document]
Comment (razvan) [reply olm/build-manifests.sh:13]: Done.

File: olm/build-manifests.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (razvan) [reply olm/build-manifests.sh]: I tried to clarify things a bit more in the comments. Ideally, we would do these replacements automatically too.

[the remark this answers is not in the document]
Comment (adwk67) [reply olm/build-manifests.sh]: Thanks!

File: olm/build-manifests.sh [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+		---
+		annotations:
+		  operators.operatorframework.io.bundle.channel.default.v1: stable
+		  operators.operatorframework.io.bundle.channels.v1: stable
+		  operators.operatorframework.io.bundle.manifests.v1: manifests/
+		  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+		  operators.operatorframework.io.bundle.metadata.v1: metadata/
+		  operators.operatorframework.io.bundle.package.v1: stackable-${OPERATOR}
+
+		  com.redhat.openshift.versions: v4.10-v4.11
```
[the remark this answers is not in the document]
Comment (razvan) [reply olm/build-manifests.sh]: Updated https://github.com/stackabletech/stackable-utils/pull/54/commits/5a819c6361ca58588cf245d4f99e61bd4a51e5aa

**Review by adwk67 — APPROVED** [review]
LGTM!

**Pull request merged**